lib/mcb/commands/geocode.rb

```diff
@@ -0,0 +1,37 @@
+summary "Batch geocode addresses"
+option :m, "model", "model to be geocoded", argument: :required, transform: method(:String)
+option :s, "sleep", "time to sleep between each request", argument: :optional, default: 0.5, transform: method(:Float)
+option :b, "batch_size", "batch size", argument: :optional, default: 100, transform: method(:Integer)
+
+run do |opts, args, _cmd| # rubocop:disable Metrics/BlockLength
+  MCB.init_rails(opts)
+
+  if opts[:model].nil?
+    raise "Please pass in the model you want to geocode"
+  end
+
+  unless %w(provider site).include? opts[:model].downcase
+    raise "You can only geocode Sites and Providers"
+  end
+
+  geocode_record = lambda { |obj|
+    obj.geocode
+    obj.save
+    sleep(opts[:sleep])
+  }
+
+  require "geocoder"
```

Comment:
Middle of nowhere?

Reply:
@defong - my understanding (which may be incorrect) is that any external libraries used within MCB should be required just before use (rather than at the top of the file) in order to optimise overall MCB load time. @misaka can you confirm?

Reply:
Ya, that's right. Although, I think it could go at the top of the `run` method here.